### ISSUE TITLE
Wish: respect `isearch-open-invisible`on n/p

### ISSUE THREAD

**Issue by jdtsmith** [oantolin/embark#571]
My favorite way to find symbols is to invoke `embark-act` followed by `embark-next|previous-symbol`.  It would be lovely if these commands respected `isearch-open-invisible` in the same manner isearch does: opening and re-hiding invisible text blocks (ala `outline-mode`) using the function named in this text property.  Could possibly just re-use isearch's capabilities here since the two shouldn't be possible to run at the same time.

**Comment by minad** [comment]
Good idea. You can use the respective cde from consult which is based on isearch/org-fold.

https://github.com/minad/consult/blob/b404039f567ce296823425b641b2348482a1750e/consult.el#L1344-1354
